awesomerc.lua.in

```diff
@@ -270,7 +276,9 @@ globalkeys = awful.util.table.join(
                   awful.util.getdir("cache") .. "/history_eval")
               end),
     -- Menubar
-    awful.key({ modkey }, "p", function() menubar.show() end)
+    awful.key({ modkey }, "p", function() menubar.show() end),
+    -- Alt + Left Shift switches the current keyboard layout
+    awful.key({ "Mod1" }, "Shift_L", function () mykeyboardlayout.next_layout() end)
```

Comment:
I disagree with `Alt+Shift` as those are 2 modifiers, it will cause problems with everybody who use Alt as their modkey (mostly former DWM users), Mac OSX (and my config) use Alt+Space. I understand Shift is used by Windows and is quite convenient to use, but it may cause a little backsplash.

Other than that, this patch now look fine to me. I will test it later today then, unless someone else complain, merge it.